.github/ISSUE_TEMPLATE/test-suite-failure.md

```diff
@@ -0,0 +1,9 @@
+---
+name: OpenJ9 test suite failure
+about: A failing functionality, system, or JTReg test
+
+---
+
+Public link to the failing test. If the link is internal, instead include which test suite (functionality, system, etc), platform, JDK version, build level.
+
+Output from the failure, including test name.
```

Comment:
We may want to wrap some of these in `<!---` and `-->` so they're displayed as comments in case the issue author forgets to delete them. We can also introduce some sections. For example:

---
### Failure link
<!--- Replace this with a link to the failing test case. If the link is internal, instead include which test suite (functionality, system, etc), platform, JDK version, build level. -->

### Output from the failure
<!--- Place the output from the failure, including test name withing the triple back-ticks below -->
```
```

Reply:
There is some risk that a user would put text inside the comments. I've added sections and introduced additional formatting.